app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -80,6 +83,7 @@ sealed class NavigationItem(
             Conversation
         )
 
+        @OptIn(ExperimentalMaterialApi::class)
```

Comment:
Is it necessary to add this annotation at this point, since we are already adding it at the top level of the class ?